eth/tracers/api.go

```diff
@@ -448,6 +463,20 @@ func (api *API) TraceBlockByHash(ctx context.Context, hash common.Hash, config *
 	if err != nil {
 		return nil, err
 	}
+
+	if api.backend.ChainConfig().IsOptimismPreBedrock(block.Number()) {
```

Comment:
```suggestion
	if api.backend.ChainConfig().IsPreKromaMPT(block.Number()) {
```
Is it your intention to use IsOptimismPreBedrock to reduce code differences?